src/ui/component/Page.html

```diff
@@ -14,6 +14,12 @@
     <script async src="https://www.googletagmanager.com/gtag/js?id={{config.GOOGLE_ANALYTICS_ID}}"></script> {{{config.GOOGLE_ANALYTICS_CODE}}}
 </head>
 <body>
+    <div id="global-cookie-message">
+        <p>
+            GOV.UK uses cookies to make the site simpler.
```

Comment:
Probably need to review this message - can you run past Rich?

Reply:
Changing to Civil Service Learning